GOVERNANCE.md

```diff
@@ -0,0 +1,132 @@
+# Jujutsu Governance
+
+## Overview
+
+Jujutsu is an open source project, lead, maintained and designed for a worldwide
+community. Anyone who is interested can join, contribute, and participate in the
+decision making process. This document is intended to help you understand how
+you can do that.
+
+## Project roles
+
+There are two broadly defined roles in the Jujutsu project: **Maintainers** and
+**Contributors**.
+
+### Maintainers
+
+At the top of the hierarchy are the **Maintainers**. We consider maintainers to
+be the people who contribute, review, guide, and collectively make decisions
+about the direction and scope of the project.
+
+A typical maintainer is not only someone who has made "large" contributions, but
+someone who has shown they are continuously committed to the project and its
+community. Some expected responsibilities of maintainers include (but are not
+exclusively limited to):
+
+- Displaying a high level of commitment to the project and its community, and
+being a role model for others.
+- Writing code &mdash; a lot of code, especially "glue code" or "grunt work";
+fixing bugs, improving processes, making judgments on dependencies, handling
+security vulnerabilities, and so on and so forth.
+- Reviewing code submitted by others &mdash; with an eye to maintainability,
+performance, code quality, and "style" (fitting in with the project).
+- Participating in design discussions, especially with regards to architecture
+or long-term vision.
+- When necessary, participate in a collective voting process to resolve
+conflicts and make decisions for which there is no clear answer.
+
+This is not an exhaustive list, nor is it intended that every maintainer does
+each and every one of these individual tasks to equal amounts. Rather this is
+only a guideline for what maintainers are expected to conceptually do.
+
+In short, Maintainers are the outwardly visible stewards of the project.
+
+#### Current list of maintainers
+
+The current list of maintainers:
+
+- Yuya Nishihara
+- Martin von Zweigbergk
+- TBD
+
+TODO: Pending on the initial process of open nominations, decided by Yuya and
+Martin, more names will be added.
+
+### Contributors
+
+We consider contributors to be active participants in the project and community
+who are *not* maintainers. These are people who might:
+
+- Help users by answering questions
+- Participating in lively and respectful discussions across various channels
+- Submit high quality bug reports
+- Submit patches or pull requests
+- Help with testing and quality assurance
+- Submit feedback about planned features, use cases, or bugs
+
+Note that we consider Contributors to all be valuable, but we *do* define them
+as people **who actively participate in the project**. Examples of things that

```

Comment:
This seems to be saying that we consider *non*-Contributors valuable. Is that what you meant? Because the "but" doesn't currently seem to indicate any contradiction, so it's a bit confusing. I'm not sure how to say it in a better way. We could just drop the part saying that we consider them valuable.